src/growth_region.h

```diff
@@ -60,66 +58,69 @@ class GrowthRegion : public cyclus::Region {
   /// enter the simulation and register any children present
   virtual void EnterNotify();
 
-  /// register a new child
-  virtual void BuildNotify(Agent* m);
-
   /// unregister a child
   virtual void DecomNotify(Agent* m);
 
   inline cyclus::toolkit::SupplyDemandManager* sdmanager() {
     return &sdmanager_;
   }
-
- protected:
-  #pragma cyclus var {"tooltip": "commodity in demand", \
-                      "doc": "name of the commodity experiencing a " \
-                             "growth in demand", \
-                      "uilabel": "Growth Commodity", \
+  
+ protected:  
+  #pragma cyclus var { \
+    "alias": ["growth", "commod", \
+              ["piecewise_function",                                    \
+               ["piece", "start", ["function", "type", "params"]]]],    \
+    "uitype": ["oneormore", "string", \
+               ["oneormore", \
+                ["pair", "int", ["pair", "string", "string"]]]],      \
+    "uilabel": "Growth Demand Curves", \
+    "doc": "Nameplate capacity demand functions." \
+    "\n\n"                                       \
+    "Each demand type must be for a commodity for which capacity can be built "\
+    "(e.g., 'power' from cycamore::Reactors). Any archetype that implements the "\
+    "cyclus::toolkit::CommodityProducer interface can interact with the "\
+    "GrowthRegion in the manner."                                       \
+    "\n\n"                                       \
+    "Demand functions are defined as piecewise functions. Each piece must "\
+    "be provided a starting time and function description. Each function "\
+    "description is comprised of a function type and associated parameters. "\
+    "\n\n"                                       \
+    "  * Start times are inclusive. For a start time :math:`t_0`, the demand "\
+    "function is evaluated on :math:`[t_0, \infty)`."                     \
```

Comment:
Because this is a python string before it is rst, does this need to be double \\?